docs/topics/modules/analyze_che_dependencies.adoc

```diff
@@ -20,11 +20,12 @@ image::pomxml.png[Pom.xml File]
 </dependency>
 ----
 +
-. An error icon appears near the line numbers when you add the XML code. Move the mouse pointer over the icon to view the error message (see screen capture).
+. An error icon appears next to the lines of code you modified. Move the mouse pointer over the icon to view the error message.
 +
 image::red_x.png[Error Icon]
 +
-The dependency analyzer displays this error to indicate that the current version (1.1.0) has a CVE issue (Common Vulnerabilities and Exposures) associated with it. It also recommends using version 1.2.1 instead to fix the problem.
-. Update the version in the *<version>* tags of the code to `1.2.1`. The red icon disappears when the version number is updated.
+The error indicates that the dependency analyzer has identified a high Common Vulnerabilities and Exposures (CVE) issue  associated with the current version, 1.1.0. It also recommends using, an alternate, secure version, 1.2.1, instead to fix the issue.
```

Comment:
-> "using an alternate" (no comma)

I think I'm commenting on this same mistake for like the third time.